### PR TITLE
Fix - text renders with default size when size is 0

### PR DESCRIPTION
**Pull request by Zeljko-Predjeskovic**

### CODE REVIEW

File: Svg/Platform/SkiaTextRenderer.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ private void RenderStroke(SvgTextBase txt, ISvgRenderer renderer)
                 ((SkiaBrushBase)brush).Paint.Typeface = SKTypeface.FromFamilyName(txt.FontFamily, txt.FontWeight.ToSkFontStyleWeight(), SKFontStyleWidth.Normal, txt.FontStyle.ToSkFontStyleSlant());
                 using (var pen = (SkiaPen)SvgEngine.Factory.CreatePen(brush, txt.StrokeWidth.Value))
                 {
-                    pen.TextSize = txt.FontSize.Value;
+                    pen.TextSize = txt.FontSize.Value != 0 ? txt.FontSize.Value : 9;
```
Comment (gentledepp) [inline Svg/Platform/SkiaTextRenderer.cs]: why `9`? Ist that some _magical number_?

File: Svg/Platform/SkiaTextRenderer.cs [hunk 2]
```diff
@@ -68,7 +68,7 @@ private void RenderFill(SvgTextBase txt, ISvgRenderer renderer)
 
                 using (var pen = (SkiaPen)SvgEngine.Factory.CreatePen(brush, 0f))
                 {
-                    pen.TextSize = txt.FontSize.Value;
+                    pen.TextSize = txt.FontSize.Value != 0 ? txt.FontSize.Value : 9;
```
Comment (gentledepp) [inline Svg/Platform/SkiaTextRenderer.cs]: again... ?

**Review by gentledepp — CHANGES_REQUESTED** [review]
Please do not use _magical numbers_ - there must be a different solution to this

File: Svg/Platform/SkiaTextRenderer.cs [hunk 3]
```diff
@@ -40,7 +40,7 @@ private void RenderStroke(SvgTextBase txt, ISvgRenderer renderer)
                 ((SkiaBrushBase)brush).Paint.Typeface = SKTypeface.FromFamilyName(txt.FontFamily, txt.FontWeight.ToSkFontStyleWeight(), SKFontStyleWidth.Normal, txt.FontStyle.ToSkFontStyleSlant());
                 using (var pen = (SkiaPen)SvgEngine.Factory.CreatePen(brush, txt.StrokeWidth.Value))
                 {
-                    pen.TextSize = txt.FontSize.Value;
+                    pen.TextSize = txt.FontSize.Value != 0 ? txt.FontSize.Value : pen.TextSize;
```
Comment (gentledepp) [inline Svg/Platform/SkiaTextRenderer.cs:43]: cool - where'd you get that from?

**Review by gentledepp — APPROVED** [review]

**Review by eszterhorvath — APPROVED** [review]

File: Svg.Editor.Core/Svg.Editor.Core.csproj [hunk 4]
```diff
@@ -4,12 +4,14 @@
     <AssemblyName>Svg.Editor</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>2.4.3.11</Version>
+    <Version>2.4.3.13</Version>
```
Comment (eszterhorvath) [inline Svg.Editor.Core/Svg.Editor.Core.csproj:7]: why 13 after 11?

**Pull request merged**